__tests__/app.test.js

```diff
@@ -125,6 +125,65 @@ describe("/articles/:article_id/comments", () => {
         });
       });
   });
+  test("POST:201 inserts a new comment of a given article_id to the db and sends the new comment back to the client for a username that is in the users table", () => {
+    const newComment = {
+      author: "butter_bridge",
+      body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+    };
+    return req(app)
+      .post("/api/articles/6/comments")
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.comment).toHaveProperty("body");
+        expect(res.body.comment).toHaveProperty("votes");
+        expect(res.body.comment).toHaveProperty("author");
+        expect(res.body.comment).toHaveProperty("created_at");
+        expect(res.body.comment).toHaveProperty("comment_id");
+        expect(res.body.comment.article_id).toEqual(6);
+      });
```

Comment:
We should add a test for when there are unnecessary properties in the comment I.E. {fruit: 'apple', username: 'butter_bridge', body: 'this is a comment'}. We want to ignore the extra keys and send back 201 and the comment 🙂